pykern/pkcli/github.py

```diff
@@ -422,23 +427,26 @@ def _repo(self, repo):
 
         def _clone(suffix):
             base = bd + suffix
-            prev = self._prev_backup(base)
-            if prev:
-                self._extract_backup(prev)
-                with pkio.save_chdir(base):
-                    _shell(["git", "remote", "update"])
-            else:
+            prev = self._prev_backup(base, ext="*")
+            if not prev:
                 _shell(
-                    [
+                    (
```

Comment:
why change this to a tuple?

Reply:
Good question: Tuples are read-only. This is the principle of least privilege. I've started making these changes when I'm in code. It's not crucial, obviously. I just think it is good practice.